.github/workflows/publish.yml

```diff
@@ -5,18 +5,17 @@ on:
     types: closed
     branches: master
     paths: 'libs/**'
-
+    
 defaults:
   run:
     shell: bash
 
 jobs:
-  bump_version_and_publish:
-    name: Bump and publish new package version to npm
+  bump_package_versions:
+    name: Bump packages on changes in the their respective libs
```

Comment:
Suggestion for shorter name: "Bump changed packages"  or "Bump relevant packages"

Reply:
We should perhaps also signal this Is where the publish to npm happens: 
"Version bump and publish changed pancakes"?

Reply:
My Mac autocorrected to pancakes. I'm not going to correct that delicious mistake 🥞